### PR TITLE
Post falling into machine learning

### PR DESCRIPTION
**Pull request by srodrigo**
Initial review was done here:
https://docs.google.com/document/d/1fMqUHGFPO8lui8ZycIe8yIXBrLNEF2d1Sfbd4jrl1rU/edit

### CODE REVIEW

File: _posts/2017-11-17-falling-into-machine-learning.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## What can I do with Machine Learning?
+Machine Learning can sound like a mysterious thing, but we actually see it every day. The amount of applications is countless:
+
+* How do Amazon or Netflix know what you like, to make recommendations?
+* What’s behind those self-driving cars?
+* Is it possible to automate financial trading?
+* What about weather forecasts?
+* How does the Google Search Engine work?
+* Can machines help us improve healthcare?
+* What made Siri or Alexa be able to process natural language?
```
Comment (samanthalcohen) [inline _posts/2017-11-17-falling-into-machine-learning.md]: Might read better if it was . "What gives Siri and Alexa the ability to process natural language?"

File: _posts/2017-11-17-falling-into-machine-learning.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+And these are only a few examples. The use cases of Machine Learning are left up to imagination.
+
+## What do I need to get started?
+It’s easy to feel overwhelmed by the amount of information that we can find about Machine Learning. There is a huge amount of blogs and papers to look at, and this grows every day, as the field is still quite young and the interest about it has increased in the last few years.
+
+As a multidisciplinary field, there are a lot of areas of Computer Science and Mathematics that you need to have at least some knowledge about.
+
+Programming is a must. As Machine Learning practitioner, you will usually be doing more hands-on work than a data scientist, who will be more focused on data analysis. There is a thin line between the two though, and you could even be performing both roles at the same time, specially when working on your own. Proficiency in at least one general-purpose programming language is needed in order to build the machine learning bits, and full applications around them for users.
+
+Machine Learning is specially heavy on maths. Just to start with, you will need to have some idea about:
```
Comment (samanthalcohen) [inline _posts/2017-11-17-falling-into-machine-learning.md]: specially -> especially: "Machine Learning is especially heavy on maths."

File: _posts/2017-11-17-falling-into-machine-learning.md (hunk 1 above)
Comment (srodrigo) [reply _posts/2017-11-17-falling-into-machine-learning.md]: It does! I struggled to express that sentence.

File: _posts/2017-11-17-falling-into-machine-learning.md (hunk 2 above)
Comment (srodrigo) [reply _posts/2017-11-17-falling-into-machine-learning.md]: 👍

**Review by cohen990 — APPROVED** [review]

**Pull request merged**